### DISCUSSION TITLE
Radiator Thermostat??

### DISCUSSION THREAD

**Discussion by Mark-Muc** [arendst/Tasmota#18150] [Ideas]
Hello .. maybe someone can help.
I'd like to use  Radiator Thermostates . I do have some AVM (Fritz) 301 .. but I'd like to have some, with i can controle with 
Tasmota / MQTT / Node-Red 
Some one have a solution? Tuya? What ever..
Thx for any help / hint
Mark

**Comment by kroon040** [comment]
With EQ3 bluetooth radiator thermostats you can control these

https://tasmota.github.io/docs/EQ3-TRV/

**Reply by Mark-Muc** [reply]
Thank u kroon040!!
I bought on for testing an it worked ... i also saw on:  https://tasmota.github.io/docs/Bluetooth_ESP32/#mi32-mqtt-messages
![image](https://user-images.githubusercontent.com/67410847/226301913-d7e514b9-fb26-458d-be34-c383d4f8875a.png)

That's still in development, but up to now . it is usable .. maybe there comes also something in the UI like Thermostat ...
Maybe also so examples how to use with the LYWSD03MMC (in generally) would be helpfull maybe also in Berry ((UI change)!!
Thx again
Mark
